### DISCUSSION TITLE
The client traffic does not pass through Strongswan

### DISCUSSION THREAD

**Discussion by wyw900807** [strongswan/strongswan#388] [Q&A]
Dear sir:
        The IP of client is 3.3.3.10, the public IP of strongswan is 4.4.4.10 and the IP of target is 5.5.5.10. After building the connection, I ping the target and find that the source IP of the packet is 3.3.3.10 and the destination IP of the packet is 5.5.5.10, it means that the packet does not pass through Strongswan，I do not know why. I think the packet sent by the client should be an ESP packet with a source address of 3.3.3.10 and a destination address of 4.4.4.10

**Comment by wyw900807** [comment]
the configuration is shown as follows:

conn windows10
    keyexchange=ikev2
    ike=aes256-sha1-modp1024!
    rekey=no
    left=%defaultroute
    leftid=4.4.4.10
    leftauth=pubkey
    leftsubnet=0.0.0.0/0
    leftcert=server.cert.pem
    right=%any
    rightauth=eap-mschapv2
    rightsourceip=10.31.3.0/24

**Reply by Thermi** [reply]
Please do it exactly as shown in #196
